packages/base/lua/CellularSpace.lua

```diff
@@ -26,6 +26,114 @@ local terralib = getPackage("terralib")
 
 TeCoord.type_ = "Coord" -- We now use Coord only internally, but it is necessary to set its type.
 
+local function loadNeighborhoodGAL(self, data)
+	local file = openFile(data.source, "r")
+	if file == nil then resourceNotFoundError("file", file) end
+
+	local header = file:read()
+	local lineTest = CSVparseLine(header, "\t" )
+	local b
+	if lineTest[2] ~= nil then b = "\t"
+	else b = " " end
+
+	forEachCell(self, function(cell)
+		cell:addNeighborhood(Neighborhood{}, data.name)
+	end)
+
+	repeat
+		local line_cell = file:read()
+		if line_cell ~= nil then
```

Comment:
it is better to declare line_cell = file:read() before the repeat and in the end and avoid this if here, in the following way:

```lua
line_cell = ...
while line_cell do
    -- process
    line_cell = ...
end
```